api/src/transactions.rs

```diff
@@ -103,8 +104,10 @@ pub struct TransactionsApi {
 impl TransactionsApi {
     /// Get transactions
     ///
-    /// Get on-chain (meaning, committed) transactions. You may specify from
-    /// when you want the transactions and how to include in the response.
+    /// Retrieve on-chain committed transactions.  The page size and start can be provided to
```

Comment:
nit: one space instead of two

Reply:
same thing for other comments

Reply:
Could you also add a comment like with the accounts endpoint like "if no start version is given, it will start at x".